spinalcordtoolbox/reports/qc.py

```diff
@@ -62,22 +63,27 @@ class QcImage(object):
                      "#a22abd", "#d58240", "#ac2aff"]
     # _seg_colormap = plt.cm.autumn
 
-    def __init__(self, qc_report, interpolation, action_list, stretch_contrast=True,
-                 stretch_contrast_method='contrast_stretching', angle_line=None):
+    def __init__(self, qc_report, interpolation, action_list, process, stretch_contrast=True,
+                 stretch_contrast_method='contrast_stretching', angle_line=None, fps=None):
```

Comment:
Ah, one other comment. Could you double check that adding the `process` parameter like this meshes with the existing usage of `QcImage`? For example:

https://github.com/neuropoly/spinalcordtoolbox/blob/916ab591ca2e35cab9f5f27ec2439ce8fdb0b9d2/spinalcordtoolbox/reports/qc.py#L565-L566

Reply:
I'm not sure what you mean here! Is it something to do with the order of the parameters? Right now I'm calling `QcImage` like this : 
https://github.com/neuropoly/spinalcordtoolbox/blob/e17b2fc4423ef94087fb10e59802e2c5fdfdfab1/spinalcordtoolbox/reports/qc.py#L693-L694

Reply:
Ah, my apologies! I was looking at master instead of this branch, and thought there was a call to `QcImage` that hadn't been changed. :facepalm: